src/components/ChatEntry.js

```diff
@@ -1,22 +1,51 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
+// import { useState } from 'react';
 
 const ChatEntry = (props) => {
+  // const [isLiked, setIsLiked] = useState(false);
+
+  // const toggleLiked = () => {
+  //   console.log('I have clicked on a heart!');
+  //   setIsLiked(!isLiked);
+  // };
+
+  const onLikeClick = () => {
+    console.log('I clicked a heart!');
+    const updatedChat = {
+      liked: !props.liked,
+    };
+    props.onUpdate(updatedChat);
+  };
+
+  const heartStatus = props.liked ? '❤️' : '🤍';
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+
+        <p>
+          <TimeStamp time={props.timeStamp} />
+        </p>
+        <button onClick={onLikeClick} className="heart-liked">
```

Comment:
It looks like the className got changed from `like` to `heart-liked`. The `like` class is used by the tests and since the name was changed, it caused one of the tests to fail. Be careful when changing class names! The name could be used to give the element a very specific CSS or functionality for a test.